src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessarysemicolonaftertypememberdeclaration/Example1.java

```diff
@@ -0,0 +1,32 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="UnnecessarySemicolonAfterTypeMemberDeclaration"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessarysemicolonaftertypememberdeclaration;
+
+// xdoc section -- start
+class Example1 {
+  ; // violation, standalone semicolon
+  {}; // violation, extra semicolon after init block
+  static {}; // violation, extra semicolon after static init block
+  Example1() {}; // violation, extra semicolon after constructor definition
+  void method() {}; // violation, extra semicolon after method definition
+  int field = 10;; // violation, extra semicolon after field declaration
+
+  {
+    ; // no violation, it is empty statement inside init block
```

Comment:
`// ok, it is empty statement inside init block`

please apply the same change to other "no violation" lines.
it is better be have "ok" and "violation" as very different words, as some people do not read "no".

Reply:
Addressed in https://github.com/checkstyle/checkstyle/pull/15736/commits/8c114736d158ff7af5664d979deb81a690cb76b0